examples/quickstart-xgboost/run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/
+
+# Download the CIFAR-10 dataset
+python -c "from dataset import init_higgs; init_higgs(20, "uniform")"
```

Comment:
Oh i forgot to update `run.sh`. i don't think we need this file as the instruction of how to run the code is in readme. I'll remove it later if agreed

Reply:
I think it's really common to have it (it's present in many examples). It's just the 
```
# Download the CIFAR-10 dataset
python -c "from dataset import init_higgs; init_higgs(20, "uniform")"
```
that might be removed and it will work well